setup/dusty.spec

```diff
@@ -0,0 +1,35 @@
+# -*- mode: python -*-
+
+import os
+
+block_cipher = None
+
+a = Analysis(['bin/dusty'],
+             pathex=[os.path.abspath('.')],
+             binaries=None,
+             datas=[],
+             hiddenimports=[],
+             hookspath=[],
+             runtime_hooks=['setup/binary-hook.py'],
+             excludes=[],
+             win_no_prefer_redirects=False,
+             win_private_assemblies=False,
+             cipher=block_cipher)
```

Comment:
I think pyinstaller uses this as an entrypoint to try to figure out what to bundle into the binary